src/main/java/com/stripe/model/StripeCollection.java

```diff
@@ -74,6 +83,10 @@ public Iterable<T> autoPagingIterable(Map<String, Object> params) {
    * @param options request options (will override the options from the initial list request)
    */
   public Iterable<T> autoPagingIterable(Map<String, Object> params, RequestOptions options) {
+    String apiKey = options.getApiKey();
+    if (apiKey == null) {
```

Comment:
Should this also check the global `Stripe.apiKey`?

Reply:
I think if `RequestOptionsBuilder` (which it should?) is used, then `options.getApiKey()` defaults to `Stripe.apiKey`

Reply:
Good call - that sounds good to me!